frontend/src/scenes/notebooks/Notebook/Notebook.tsx

```diff
@@ -99,9 +97,7 @@ export function Notebook({ shortId, editable = false, initialAutofocus = null }:
                 ) : null}
 
                 <div className="flex flex-1 justify-center space-x-2">
```

Comment:
I don't know if this was always the case but the space-x-2 causes a gap when there are no widgets showing... (see the snapshot diff)